tests/e2e/tests/test_football_prediction_market.py

```
@@ -0,0 +1,113 @@
+    # Handle assertion style differences
+    assert tx_execution_succeeded(resolve_receipt)
+
+    # Get Updated State
+    contract_state_2 = client.read_contract(
+        address=contract_address,
+        function_name="get_resolution_data",
+    )
+
+    assert contract_state_2["winner"] == 1
+    assert contract_state_2["score"] == "2:0"
+    assert contract_state_2["has_resolved"] == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if contract_state_2["has_resolved"]:` for truth checks

Replace with `contract_state_2["has_resolved"]`

(E712)